src/module.rs

```diff
@@ -74,6 +74,20 @@ impl<'a> Module<'a> {
         self.segments.last_mut().unwrap()
     }
 
+    /// Should config exists, get a reference to a newly created segment in the module
```

Comment:
Sorry, I'm not 100% clear on why this function exists. Is this just creating a segment only if it's in the config file?

Reply:
I guess clarifying a little more, I'm just confused about the naming and comments for this function. The actual functionality is pretty clear from reading the code.

Reply:
Hmm, sorry I can't find a better naming. Should we name it `new_segment_if_config_exists` or something?

Reply:
@heyrict I think that would be a good name.